packages/xpetra/src/Map/Xpetra_EpetraMap.cpp

```diff
@@ -49,6 +49,12 @@
 
 #include "Xpetra_EpetraMap.hpp"
 
+#ifdef HAVE_XPETRA_KOKKOS_REFACTOR
+#ifdef HAVE_XPETRA_TPETRA
+#include "Tpetra_Details_FixedHashTable_def.hpp"
```

Comment:
:( This is kinda ungood.  If you need another instantiation, add another instantiation.  I can show you how.

Reply:
Also, this could cause link errors due to multiple definitions, depending on Tpetra's ETI settings.

Reply:
I agree with Mark.  I also don't like the fact that there's a dependence on Tpetra in the Epetra path.